<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogArtista</class>
 <widget class="QDialog" name="DialogArtista">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pbAgregar">
      <property name="text">
       <string>agregar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbMod">
      <property name="text">
       <string>modificar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>22</y>
     <width>249</width>
     <height>180</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Fecha descubrimiento</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Peso</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Titulo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Tecnica</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Tama√±o</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Tipo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="leFecha"/>
      </item>
      <item>
       <widget class="QLineEdit" name="leNombre"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lePeso"/>
      </item>
      <item>
       <widget class="QLineEdit" name="leTitulo"/>
      </item>
      <item>
       <widget class="QLineEdit" name="leTecnica"/>
      </item>
      <item>
       <widget class="QLineEdit" name="leTamanio"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Pintura</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Escultura</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dibujo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bosquejo</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
